lang/en/docs/tutorials/python-ml/train-clustering-model.md

```diff
@@ -0,0 +1,152 @@
+# Train a K-Means Clustering Model with Scikit-Learn
+
+This tutorial demonstrates how to train a K-Means Clustering [^1] model using Scikit-Learn. [^2]
+
+## 1. Acquire Training Data
+
+Unsupervised learning [^3] is a technique that takes in unlabeled training data, and generates its own labels for a
+dataset. Oftentimes, it is used as an exploratory tool to find collections of similar items. For example, to find
+molecules or crystals with similar properties.
+
+The data used in this example was acquired from Kaggle: [Here](https://www.kaggle.com/anlgrbz/super-conductors)
```

Comment:
same as https://github.com/Exabyte-io/documentation/pull/192/files#r619504615

Reply:
We do this a few lines down - updated the docs so that this is now a reference, to make it easier to spot where to get the data.